### ISSUE TITLE
Group missing from "3A Repair Limpet Controller" entry in JSON export

### ISSUE THREAD

**Issue by psdevil** [EDCD/coriolis#671]
Think I may have spotted a bug in the JSON export of the builds.

The size "3A Repair Limpet Controller" entry (optional internal slot 5 in my AspEx build) does not have a "group" field associated with it which seems to be the name of the installed module. This is also true for optional internal slot 4 in my Phantom build and a new DBX build I started. 

It would appear to be related to the module entry and not the position in the output.

This snippet is from my Phantom build. You will see the issue in the 4th entry.

I was also wondering why some entries have names and group names, while others just have group names. Is this a bug too?

I am a dev myself, but I have never used Javascript, otherwise I would have delved deeper for this issue report.

```
    "internal": [
      {
        "class": 6,
        "rating": "A",
        "enabled": true,
        "priority": 1,
        "group": "Fuel Scoop"
      },
      {
        "class": 5,
        "rating": "A",
        "enabled": true,
        "priority": 3,
        "group": "Auto Field-Maintenance Unit"
      },
      {
        "class": 5,
        "rating": "D",
        "enabled": true,
        "priority": 1,
        "group": "Shield Generator"
      },
      {
        "class": 3,
        "rating": "A",
        "enabled": true,
        "priority": 4
      },
      {
        "class": 3,
        "rating": "E",
        "enabled": true,
        "priority": 1,
        "group": "Cargo Rack"
      },
      {
        "class": 1,
        "rating": "E",
        "enabled": true,
        "priority": 1,
        "group": "Supercruise Assist",
        "name": "Supercruise Assist"
      },
      {
        "class": 2,
        "rating": "G",
        "enabled": true,
        "priority": 3,
        "group": "Planetary Vehicle Hangar"
      },
      {
        "class": 1,
        "rating": "E",
        "enabled": true,
        "priority": 1,
        "group": "Docking Computer",
        "name": "Advanced Docking Computer"
      },
      {
        "class": 1,
        "rating": "I",
        "enabled": true,
        "priority": 1,
        "group": "Surface Scanner",
        "name": "Detailed Surface Scanner"
      }
    ]
  },
,
```

**Comment by Sid127** [comment]
@felixlinker this can be closed

**Comment by felixlinker** [comment]
The PR references are off between this issue and #701, right? Seems like this issue got fixed by https://github.com/EDCD/coriolis-data/pull/90 and #701 got fixed by #708
